crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/config/SpringCloudAwsConfig.java

```diff
@@ -22,4 +27,15 @@ public MappingJackson2MessageConverter createMappingJackson2MessageConverter() {
         return messageConverter;
     }
 
-}
\ No newline at end of file
+    @Bean
+    public ErrorHandler<Object> errorHandler() {
+        return new ErrorHandler<>() {
+            @Override
+            public void handle(Message<Object> message, Throwable t) {
+                deadLetterMessageService.logDeadLetterMessage(
+                    ERROR_HANDLING_MESSAGE,
+                    message.getPayload().toString());
+            }
+        };
+    }
```

Comment:
Thanks, tested and concluded it is no longer required, now removed 👍 The reason I moved away from using this exception handling by the way is because there is no way to get the actual reason for the failure - the throwable is of type _CompletionException_ and does not contain the validation message.